src/planners/PlanningTypes.cpp

```diff
@@ -0,0 +1,72 @@
+//
+// Created by nermin on 16.03.24.
+//
+
+#include "PlanningTypes.h"
+
+namespace planning
+{
+	std::ostream &operator<<(std::ostream &os, const planning::PlannerType &type) 
+	{
+		switch (type)
```

Comment:
Don't you have an unordered_map for this?

Reply:
Yes, but I can only access to planning::PlannerType by string, and not vice versa, because I created std::unordered_map<std::string, planning::PlannerType>. If I create std::unordered_map<planning::PlannerType, std::string>, it will make other changes...